packages/autocomplete/src/index.ts

```diff
@@ -199,16 +216,18 @@ function isAcceptable(
 }
 
 // stage completions based on current stage string.
-function getStageAccumulators(params: AutocompleteParameters, stage: string): typeof ACCUMULATORS {
-  if (stage !== '') return [];
-
+function getStageAccumulators(
+  params: AutocompleteParameters,
+  stage: string
+): readonly typeof ACCUMULATORS[number][] {
   if (stage.includes(PROJECT)) {
-    return ACCUMULATORS.filter((acc: any) => {
+    return ACCUMULATORS.filter((acc) => {
       return isAcceptable(params, acc, 'projectVersion');
     });
   } else if (stage.includes(GROUP)) {
     return ACCUMULATORS;
   }
+  return [];
 }
```

Comment:
I think this method was not really doing what's expected before which was highlighted when `typeof ACCUMULATORS` actually became an array instead of `any`, but maybe I misunderstood what's supposed to happen there